From 5ff5eae8ee48d8f4623bd05601e4046611b2d452 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 8 Nov 2023 15:29:25 +0100
Subject: [PATCH 0628/1141] arm64: dts: st: avoid scmi regulator yaml warning
 for stm32mp251

Avoid an SCMI YAML code checking error.
The SCMI YAML oblige the node name to be regulator@xx.

Change-Id: I846aaf6c7b6cb6faf144770c680fa6e18030f604
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/338480
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 48 +++++++++++++-------------
 1 file changed, 24 insertions(+), 24 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -124,46 +124,46 @@
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					scmi_vdd33ucpd: voltd-vdd33ucpd {
-						voltd-name = "vdd33ucpd";
-						reg = <VOLTD_SCMI_UCPD>;
-						regulator-name = "vdd33ucpd";
-					};
-					scmi_vdd33usb: voltd-vdd33usb {
-						voltd-name = "vdd33usb";
-						reg = <VOLTD_SCMI_USB33>;
-						regulator-name = "vdd33usb";
-					};
-					scmi_vdda18adc: voltd-vdda18adc {
-						voltd-name = "vdda18adc";
-						reg = <VOLTD_SCMI_ADC>;
-						regulator-name = "vdda18adc";
-					};
-					scmi_vddgpu: voltd-vddgpu {
-						voltd-name = "vddgpu";
-						reg = <VOLTD_SCMI_GPU>;
-						regulator-name = "vddgpu";
-					};
-					scmi_vddio1: voltd-vddio1 {
+					scmi_vddio1: regulator@0 {
 						voltd-name = "vddio1";
 						reg = <VOLTD_SCMI_VDDIO1>;
 						regulator-name = "vddio1";
 					};
-					scmi_vddio2: voltd-vddio2 {
+					scmi_vddio2: regulator@1 {
 						voltd-name = "vddio2";
 						reg = <VOLTD_SCMI_VDDIO2>;
 						regulator-name = "vddio2";
 					};
-					scmi_vddio3: voltd-vddio3 {
+					scmi_vddio3: regulator@2 {
 						voltd-name = "vddio3";
 						reg = <VOLTD_SCMI_VDDIO3>;
 						regulator-name = "vddio3";
 					};
-					scmi_vddio4: voltd-vddio4 {
+					scmi_vddio4: regulator@3 {
 						voltd-name = "vddio4";
 						reg = <VOLTD_SCMI_VDDIO4>;
 						regulator-name = "vddio4";
 					};
+					scmi_vdd33ucpd: regulator@5 {
+						voltd-name = "vdd33ucpd";
+						reg = <VOLTD_SCMI_UCPD>;
+						regulator-name = "vdd33ucpd";
+					};
+					scmi_vdd33usb: regulator@6 {
+						voltd-name = "vdd33usb";
+						reg = <VOLTD_SCMI_USB33>;
+						regulator-name = "vdd33usb";
+					};
+					scmi_vdda18adc: regulator@7 {
+						voltd-name = "vdda18adc";
+						reg = <VOLTD_SCMI_ADC>;
+						regulator-name = "vdda18adc";
+					};
+					scmi_vddgpu: regulator@8 {
+						voltd-name = "vddgpu";
+						reg = <VOLTD_SCMI_GPU>;
+						regulator-name = "vddgpu";
+					};
 				};
 			};
 		};
